test/cbmc/proofs/MQTT_Connect/MQTT_Connect_harness.c

```diff
@@ -26,7 +26,7 @@
 #include "core_mqtt.h"
 #include "mqtt_cbmc_state.h"
 
-void harness()
+void MQTT_Connect_harness()
```

Comment:
Why change the harness names?   This is inconsistent with the starter kit templates in the master branch of aws-templates-for-cbmc-proofs.

Reply:
This is how all the other projects do it...s2n makes the name of the harness the same as the name of the file.

The real reason: I would like every proof to have a "name" for litani to refer to it by. (The argument to the `--pipeline-name` flag). So in the starter kit, the pipeline name is set to `$(HARNESS_ENTRY)`. That's how the table looks [like this](https://d2s5j4r0gkya4b.cloudfront.net/5d596477-4746-4cb2-9b6e-4a66621b5616/index.html), the rows are the harness entry names.

So: currently we have to have it this way, because that's what's in Makefile.common. If you don't like this, we should change it in the starter kit, but I recommend that we keep it this way in MQTT for now. Suggest we discuss this offline.

Reply:
You misunderstand the starter kit.  The Makefile under template-for-proof specifies:

```

Reply:
Furthermore, we are doing out of source builds, so there is a unique pathname referring to the proof harness source code itself.

Reply:
I've reverted this change as discussed offline, and introduced a new variable for the unique name of the proof. Thank you!